addons/purchase/static/src/components/monetary_field_no_zero/monetary_field_no_zero.js

```diff
@@ -0,0 +1,28 @@
+/** @odoo-module **/
+
+import {monetaryField, MonetaryField} from "@web/views/fields/monetary/monetary_field";
```

Comment:
```suggestion
import { monetaryField, MonetaryField } from "@web/views/fields/monetary/monetary_field";
```
Please follow the [linter rules.](https://github.com/odoo/odoo/wiki/Javascript-coding-guidelines#tooling)